src/ios/CDVRadar.m

```diff
@@ -215,6 +215,65 @@ - (void)startTrackingCustom:(CDVInvokedUrlCommand *)command {
+        NSNumber *originLongitudeNumber = originDict[@"longitude"];
+        double originLatitude = [originLatitudeNumber doubleValue];
+        double originLongitude = [originLongitudeNumber doubleValue];
+        CLLocation *origin = [[CLLocation alloc] initWithCoordinate:CLLocationCoordinate2DMake(originLatitude, originLongitude) altitude:-1 horizontalAccuracy:5 verticalAccuracy:-1 timestamp:[NSDate date]];
+        NSDictionary *destinationDict = optionsDict[@"destination"];
+        NSNumber *destinationLatitudeNumber = destinationDict[@"latitude"];
+        NSNumber *destinationLongitudeNumber = destinationDict[@"longitude"];
+        double destinationLatitude = [destinationLatitudeNumber doubleValue];
+        double destinationLongitude = [destinationLongitudeNumber doubleValue];
+        CLLocation *destination = [[CLLocation alloc] initWithCoordinate:CLLocationCoordinate2DMake(destinationLatitude, destinationLongitude) altitude:-1 horizontalAccuracy:5 verticalAccuracy:-1 timestamp:[NSDate date]];
+        NSString *modeStr = optionsDict[@"mode"];
+        RadarRouteMode mode = RadarRouteModeCar;
+        if ([modeStr isEqualToString:@"FOOT"] || [modeStr isEqualToString:@"foot"]) {
+            mode = RadarRouteModeFoot;
+        } else if ([modeStr isEqualToString:@"BIKE"] || [modeStr isEqualToString:@"bike"]) {
+            mode = RadarRouteModeBike;
+        } else if ([modeStr isEqualToString:@"CAR"] || [modeStr isEqualToString:@"car"]) {
+            mode = RadarRouteModeCar;
+        }
+        NSNumber *stepsNumber = optionsDict[@"steps"];
+        int steps;
+        if (stepsNumber != nil && [stepsNumber isKindOfClass:[NSNumber class]]) {
+            steps = [stepsNumber intValue];
+        } else {
+            steps = 10;
+        }
+        NSNumber *intervalNumber = optionsDict[@"interval"];
+        double interval;
+        if (intervalNumber != nil && [intervalNumber isKindOfClass:[NSNumber class]]) {
+            interval = [intervalNumber doubleValue];
+        } else {
+            interval = 1;
+        }
+
+        [Radar mockTrackingWithOrigin:origin destination:destination mode:mode steps:steps interval:interval completionHandler:^(RadarStatus status, CLLocation *location, NSArray<RadarEvent *> *events, RadarUser *user) {
+            NSMutableDictionary *dict = [NSMutableDictionary new];
+            [dict setObject:[Radar stringForStatus:status] forKey:@"status"];
+            if (location) {
+                [dict setObject:[Radar dictionaryForLocation:location] forKey:@"location"];
+            }
+            if (events) {
+                [dict setObject:[RadarEvent arrayForEvents:events] forKey:@"events"];
+            }
+            if (user) {
+                [dict setObject:[user dictionaryValue] forKey:@"user"];
+            }
+
+            CDVPluginResult* pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsDictionary:dict];
```

Comment:
oh, I see - Cordova might send back a 200, but our nested `status` in the response object will indicate the API response code